mentorship ios/Models/RelationModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  RelationModel.swift
+//  Created on 02/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+class RelationModel {
+    
+    // MARK: - Variables
+    let currentRelation = HomeModel.HomeResponseData.RequestStructure(id: 0, actionUserID: 0, mentor: nil, mentee: nil, acceptDate: 0, startDate: 0, endDate: 0, notes: "")
```

Comment:
nit: (Readabilty) This does not throw lint warning? Big line? Could break it up.

Reply:
Also, does the home model response contain the relation data or is there a separate endpoint available for just the relation data ? In which case can we use that and separate out HomeModel?

Reply:
Suggestion: We can have a RelationModel use that for both Home and Relation? Since Data struct is same.

Reply:
> Also, does the home model response contain the relation data or is there a separate endpoint available for just the relation data ? In which case can we use that and separate out HomeModel?

Home Model response contains it. The dashboard api returns all this data.

> Suggestion: We can have a RelationModel use that for both Home and Relation? Since Data struct is same.

The relation model is slightly different. However, the task has been separated out already and reused in both home model and relation model. The request structure is more heavily used in the home model, and has a single instance in relation model, and hence has been kept inside home model.